hnn_core/extracellular.py

```diff
@@ -435,10 +435,6 @@ def plot_lfp(self, *, trial_no=None, contact_no=None, tmin=None, tmax=None,
             Trial number(s) to plot
         contact_no : int | list of int | slice
             Electrode contact number(s) to plot
-        tmin : float | None
-            Start time of plot in milliseconds. If None, plot entire
-            simulation.
-        tmax : float | None
             End time of plot in milliseconds. If None, plot entire simulation.
```

Comment:
Missed this line as well.

Reply:
Oh, really sorry I missed that. Removed it now